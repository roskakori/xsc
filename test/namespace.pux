<?xml version="1.0" encoding="utf-8"?>
<!-- A simple list of all customers using namespaced tags. -->
<some:customers xmlns:some="http://example.com/some.xsd">
  <some:person/>
  <?pux for customers?>
  <some:person>
    <surname>${customers.surname}</surname>
    <firstname>${customers.surname}</firstname>
    <dateOfBirth>${customers.dateOfBirth}</dateOfBirth>
  </some:person>
  <?pux end for?>
</some:customers>
