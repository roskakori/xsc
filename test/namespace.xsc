<?xml version="1.0" encoding="utf-8"?>
<!-- A simple list of all customers using namespaced tags. -->
<some:customers xmlns:some="http://example.com/some.xsd">
  <some:person/>
  <?xsc for customers?>
  <some:person>
    <surname>${customers.surname}</surname>
    <firstname>${customers.surname}</firstname>
    <dateOfBirth>${customers.dateOfBirth}</dateOfBirth>
  </some:person>
  <?xsc end for?>
</some:customers>
