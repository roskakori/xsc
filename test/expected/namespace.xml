<?xml version="1.0" encoding="utf-8"?><!-- A simple list of all customers using namespaced tags. --><some:customers xmlns:some="http://example.com/some.xsd">
  <some:person/>
  
  <some:person>
    <surname>Doe</surname>
    <firstname>Doe</firstname>
    <dateOfBirth>1957-03-08</dateOfBirth>
  </some:person>
  
  <some:person>
    <surname>Miller</surname>
    <firstname>Miller</firstname>
    <dateOfBirth>1946-10-04</dateOfBirth>
  </some:person>
  
  <some:person>
    <surname>Webster</surname>
    <firstname>Webster</firstname>
    <dateOfBirth>1974-12-23</dateOfBirth>
  </some:person>
  
</some:customers>